src/QueryNode/QueryForm.jsx

```diff
@@ -9,14 +12,26 @@ class QueryForm extends React.Component {
   };
 
   constructor(props) {
+    const queryParams = querystring.parse(location.search ? location.search.replace(/^\?+/, '') : '');
     super(props);
     this.state = {
-      selectValue: null,
+      selectValue: Object.keys(queryParams).length > 0 && queryParams.node_type ?
+        { value: queryParams.node_type, label: queryParams.node_type }
+        : null,
     };
     this.updateValue = this.updateValue.bind(this);
+    this.handleDownloadAll = this.handleDownloadAll.bind(this);
     this.handleQuerySubmit = this.handleQuerySubmit.bind(this);
   }
 
+  handleDownloadAll() {
+    const programProject = this.props.project.split('-');
+    window.open(
+      `${submissionApiPath}${programProject[0]}/${programProject[1]}/export?node_label=${this.state.selectValue.value}&format=tsv`,
```

Comment:
the project code could contain dashes, could you add this? https://github.com/uc-cdis/data-portal/blob/master/src/utils.js#L6-L8

Reply:
:O thanks!